scenes/power_up.tscn

```diff
@@ -0,0 +1,49 @@
+[gd_scene load_steps=4 format=1]
```

Comment:
All other scenes are saved in the `.xml` format. Should we change to `.tscn` or stay in `.xml`? Doesn't make much difference, it would be just for consistency's sake.